modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -516,7 +516,7 @@ class CV_EXPORTS_W GFTTDetector : public Feature2D
 {
 public:
     CV_WRAP static Ptr<GFTTDetector> create( int maxCorners=1000, double qualityLevel=0.01, double minDistance=1,
-                                             int blockSize=3, bool useHarrisDetector=false, double k=0.04 );
+                                             int blockSize=3, int gradiantSize=3, bool useHarrisDetector=false, double k=0.04 );
```

Comment:
We can't change the public API due to binary compatibility obligations. You can create an overloaded version of the function.